bot/information/information.py

```diff
@@ -59,6 +53,14 @@ def information(
                 f"{cycle} * {number}"
             )
     elif not is_experiment:
+
+        c_rate = round(
+            params[0]["Current function [A]"]
+            / params[0]["Nominal cell capacity [A.h]"],
+            2
+        )
+        temp = round(params[0]["Ambient temperature [K]"] - 273.15, 2)
```

Comment:
I think it still makes sense to change the temperature in the experiment

Reply:
The temperature is still changing in the experiment but it is not being added to the tweet text, as it would be visible on the GIF's legend. These statements here are for constant discharge plots where the legend would have the model name and the tweet text will have a single value for varied temperature and c-rate

Reply:
Ah ok, thanks!

Reply:
I'll add comments to make it more clear